dev/build.example.testcontainers_fat/build.gradle

```diff
@@ -15,9 +15,22 @@
  // This bundle is required both at compiletime and runtime.
  addRequiredLibraries.dependsOn copyTestContainers
  
+ //TODO temporary while we switch from ojdbc8_g to ojdbc8
```

Comment:
If the code that references the new Oracle JDBC driver is all introduced at the same time, why is a temporary solution needed?  There has to be some lag the temporary solution covers., but I don't understand why it is needed.